tests/phpunit/includes/testcase-preload-paths.php

```diff
@@ -16,21 +16,12 @@ public function set_up() {
 		register_taxonomy( 'trtax', 'trcpt', array( 'show_in_rest' => true ) ); // Translated custom taxonomy.
 		register_taxonomy( 'trtax_with_no_namespace', 'trcpt', array( 'show_in_rest' => true, 'rest_namespace' => null ) ); // Translated custom taxonomy with namespace explicitly set to null as in CPTUI.
 
-		$options = array_merge(
-			PLL_Install::get_default_options(),
+		$options = self::create_reset_options();
```

Comment:
I see you're doing such a thing in many places. Given that, could `self::create_reset_options()` accept an array of options to merge with the default ones directly?

Reply:
Done in [37ee8c9](https://github.com/polylang/polylang/pull/1481/commits/37ee8c9e8a61d2efd62e24dd25c76c47e1f3333c).